fleet/fleet.go

```diff
@@ -217,3 +256,6 @@ func (f fleet) ipFromUnitState(unitState *schema.UnitState) (net.IP, error) {
 
 	return nil, maskAny(ipNotFoundError)
 }
+
+// TODO
```

Comment:
Ok I presume you mean the PrintStatus method. Do you have an alternative suggestion or at least reasons why not to put it here? Your blank statement makes it hard to act on it.